src/sinks/loki/config.rs

```diff
@@ -93,6 +97,12 @@ pub struct LokiConfig {
     /// Note: If the set of labels has high cardinality, this can cause drastic performance issues
     /// with Loki. To prevent this from happening, reduce the number of unique label keys and
     /// values.
+    #[configurable(metadata(
+        docs::examples = "vector",
+        docs::examples = "{{ event_field }}",
+        docs::examples = "{{ kubernetes.pod_labels }}",
```

Comment:
```suggestion
        docs::examples = "{{ kubernetes.pod_labels }}",
        docs::examples = "",
```

This is an awkward one for examples... would it be better/possible to represent these as the actual Key/Values rather than just Values?

```yaml
labels:
  sender: "vector"
  templated: "{{ event_field }}"
  pod_labels_*: "{{ kubernetes.pod_labels }}"
```

Reply:
It would be better, yes, but `#[configurable(metadata(...))]` doesn't yet support free-form `serde_json`-capable values to be passed, so we're stuck with string literals for examples.

Reply:
Oh, issue tracking this support yet?

Reply:
I opened https://github.com/vectordotdev/vector/issues/15547, although there's also a TODO blurb in `vector-config/src/lib.rs`, too.